dll/win32/shell32/lang/ro-RO.rc

```diff
@@ -10,16 +10,16 @@ LANGUAGE LANG_ROMANIAN, SUBLANG_DEFAULT
 
 MENU_001 MENUEX
 BEGIN
-    MENUITEM "Picto&grame mari", FCIDM_SHVIEW_BIGICON
-    MENUITEM "&Pictograme mici", FCIDM_SHVIEW_SMALLICON
-    MENUITEM "&Listă", FCIDM_SHVIEW_LISTVIEW
-    MENUITEM "&Detalii", FCIDM_SHVIEW_REPORTVIEW
+    MENUITEM "Dal&e", FCIDM_SHVIEW_BIGICON
+    MENUITEM "&Miniaturi", FCIDM_SHVIEW_SMALLICON
+    MENUITEM "&Listare", FCIDM_SHVIEW_LISTVIEW
+    MENUITEM "D&etalii", FCIDM_SHVIEW_REPORTVIEW
     MENUITEM "", -1, MFT_SEPARATOR
-    POPUP "&Ordonare după", FCIDM_SHVIEW_ARRANGE
+    POPUP "Aranjare &pictograme după", FCIDM_SHVIEW_ARRANGE
```

Comment:
It must be changed to Aran&jare according to your MS pic. The pressure on the P is immense. Many controls tend to crave that, so I am glad that you don't have to assign it here. Please change, and please notice, that there is a 2nd Aranjare a few lines below.

Reply:
No, not here! Here is the proof!

![2024-03-28 01_08_21-](https://github.com/reactos/reactos/assets/113553683/68235f04-530a-415c-8be2-7bf70df0119e)

Reply:
Ok, fine then. Easy to get confused in those menu-layouts with the multiple hierarchies. Resolved then.